frontend/test/storybook/visual-regression/v-footer.spec.ts

```diff
@@ -26,6 +27,7 @@ test.describe("VFooter", () => {
           await page.goto(pageUrl(dir, footerKind))
           if (dir === "rtl") {
             await page.locator("#language").selectOption({ value: "ar" })
+            await sleep(500)
```

Comment:
Rather than sleeping an arbitrary amount, is there a meaningful locator we `waitFor`? https://playwright.dev/docs/api/class-locator#locator-wait-for

Reply:
In the search type test failures, you can see that the button didn't display the icon. When you look into the HTML markup, you can see the `svg` in there, however, the inner `use` doesn't have the required SVG. I couldn't find a good way to wait for _that_ svg.

Reply:
I see... if the delay in loading is due to a network request, I wonder if we can [wait for the network request](https://playwright.dev/docs/api/class-page#page-wait-for-response)? I'll give this (and other similar APIs) a try tomorrow when I am working, if you don't get to it first.

My main concern with adding the sleeps is that they are inherently flaky, because if the page suddenly got _faster_, these tests would again start to fail for the same reason. On top of that, I'd have a hard time knowing how to communicate why these sleeps are in the code and when we would consider it correct to use sleeps. Of course, if there's no other way, then that's what it is. I suppose in the worst case, if we wanted to avoid sleeps at all cost, we could wait for a snapshot of the SVG element to match :confused:; If what we're concerned about is that the SVG is _visually_ loaded, that might even be the most "intelligible" way, in that it gets at the precise thing we care about waiting for.